.github/workflows/zxc-build-library.yaml

```diff
@@ -80,8 +80,14 @@ jobs:
       - name: Ensure Binary Cache Path Exists
         run: mkdir -p "${{ github.workspace }}/b/vcpkg_cache"
 
+      - name: CMake Android Build
+        if: github.event.pull_request.merged == true
+        run: |
+          docker build -t ubuntu-jammy-hedera-protobufs-cpp-android-build . --build-arg BRANCH_NAME=${{ github.head_ref }}
+          docker cp $(docker create ubuntu-jammy-hedera-protobufs-cpp-android-build:latest):/hedera-protobufs-cpp/package .
+
       - name: Install CMake & Ninja
-        uses: lukka/get-cmake@v3.27.7
+        uses: lukka/get-cmake@latest
```

Comment:
Recently experienced problems with `cmake@latest` but since nothing is failing in CI here I guess it is LGTM. Do you agree that maybe we can use a specific cmake version and not always latest as this may lead to some instability?

Reply:
Ok, I can agree with that!